app/scripts/lib/AddressBookPetnamesBridge.ts

```diff
@@ -68,18 +81,15 @@ export class AddressBookPetnamesBridge extends AbstractPetnamesBridge {
    */
   protected updateSourceEntry(type: ChangeType, entry: PetnameEntry): void {
     if (type === ChangeType.DELETED) {
-      // TODO: Replace `any` with type
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      this.#addressBookController.delete(entry.variation as any, entry.value);
+      this.#addressBookController.delete(
+        entry.variation as Hex,
+        entry.value as string,
```

Comment:
```suggestion
        entry.value,
```
entry.value is already a type string, so we won't need to do type assertion here.

Reply:
Gotcha !

Reply:
I got error as 
```
app/scripts/lib/AddressBookPetnamesBridge.ts:88:42 - error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type '`0x${string}`'.
  Type 'undefined' is not assignable to type '`0x${string}`'.

88       this.#addressBookController.delete(entry.variation, entry.value);
                                            ~~~~~~~~~~~~~~~

app/scripts/lib/AddressBookPetnamesBridge.ts:90:64 - error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type '`0x${string}` | undefined'.
  Type 'string' is not assignable to type '`0x${string}`'.

90       this.#addressBookController.set(entry.value, entry.name, entry.variation);
                                                                  ~~~~~~~~~~~~~~~
```
And it seems like coming from 
```
export type PetnameEntry = SetNameRequest & {
  // Name cannot be null in a PetnameEntry, as opposed to in a SetNameRequest,
  // where a null name indicates deletion.
  name: string;
};
```

I would prefer not touching higher level type and keep this type. I'm not a typescript expert, but since it was there as a TODO, i believe we need to add type instead of removing it.

Reply:
entry.variation does need the type assertion` as Hex` in the delete function and `as Hex | undefined` in the set function, but `entry.value` and entry.name are already string types, so they do not need the type assertion to string.

Reply:
gotcha, all fixed now, thank you !